content/applications/inventory_and_mrp/quality/quality_check_types/measure_check.rst

```diff
@@ -0,0 +1,180 @@
+
+.. image:: measure_check/measure-check-form-1.png
+   :align: center
+   :alt: A quality check form configured for a Measure quality check.
+
+Quality control point (QCP)
+---------------------------
+
+To create a |QCP| that will generate *Measure* quality checks automatically, navigate to
+:menuselection:`Quality --> Quality Control --> Control Points`, and click :guilabel:`New`. Fill out
+the new |QCP| form as follows:
+
+- In the :guilabel:`Type` drop-down field, select the :guilabel:`Measure` quality check type. Doing
+  so causes two new fields to appear: :guilabel:`Norm` and :guilabel:`Tolerance`.
+
+  - Use the first text-entry field of the :guilabel:`Norm` field to record the ideal measurement
+    that the product should conform to. Use the second text-entry field to specify the unit of
+    measurement that should be used.
+  - The :guilabel:`Tolerance` field features two sub-fields: :guilabel:`from` and :guilabel:`to`.
+    Use the :guilabel:`from` field to specify the minimum acceptable measurement, and the
+    :guilabel:`to` field to specify the maximum acceptable measurement.
+
+- In the :guilabel:`Team` drop-down field, select the quality team responsible for managing the
+  checks created by the |QCP|.
+- In the :guilabel:`Instructions` text field, enter instructions for how the measurement should be
+  taken.
+
+.. image:: measure_check/measure-check-qcp-form.png
+   :align: center
+   :alt: A QCP form configured to create Measure quality checks.
+
+Process a Measure quality check
+===============================
+
+Once created, there are multiple ways that *Measure* quality checks can be processed. If a quality
+check is assigned to a specific inventory, manufacturing, or work order, the check can be processed
+on the order itself. Alternatively, a check can be processed from the check's page.
+
+From the check's page
+---------------------
+
+To process a *Measure* quality check from the check's page, begin by navigating to
+:menuselection:`Quality --> Quality Control --> Quality Checks`, and select a quality check. Follow
+the :guilabel:`Instructions` for how to take the measurement.
+
+After taking the measurement, record the value in the :guilabel:`Measure` field on the quality check
+form.
+
+On an order
+-----------
+
+To process a *Measure* quality check on an order, select a manufacturing order or inventory order
+(receipt, delivery, return, etc.), for which a check is required. Manufacturing orders can be
+selected by navigating to :menuselection:`Manufacturing --> Operations --> Manufacturing Orders`,
+and clicking on an order. Inventory orders can be selected by navigating to
+:menuselection:`Inventory`, clicking the :guilabel:`# To Process` button on an operation card, and
+selecting an order.
+
+On the selected manufacturing or inventory order, a purple :guilabel:`Quality Checks` button appears
+at the top of the page. Click the button to open the :guilabel:`Quality Check` pop-up window, which
```

Comment:
Totally optional, minor wording change to refer to a something specific on the Odoo UI
```suggestion
at the top of the form. Click the button to open the :guilabel:`Quality Check` pop-up window, which
```

Reply:
Optionally, we could provide an image to help readers visualize where they'd see that purple "Quality Checks" button :slightly_smiling_face: 
![image](https://github.com/odoo/documentation/assets/23611711/cbbcfae3-4b7a-4780-b574-216c4ebe6efc)